packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md

```diff
@@ -7,7 +7,7 @@ Some real-world applications of the component include copying, uploading, or mov
 
 The Announced component currently has the following documented use cases:
 
-1. [Quick Actions](#/controls/web/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
-2. [Search Results](#/controls/web/announced/searchresults): Appearance of search results such as in contact fields or search boxes.
-3. [Lazy Loading](#/controls/web/announced/lazyloading): "Lazy loading" of page sections that do not appear all at once.
-4. [Bulk Operations](#/controls/web/announced/bulkoperations): Operations that require multiple sub operations, such as the moving of several files.
+1. [Quick Actions](https://developer.microsoft.com/en-us/fabric#/controls/web/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
```

Comment:
Are the hash links not working in some particular context?

Reply:
Yes. We basically are using the already existing markdown files that are shipped with OUFR package for the HIG website. In the context of the website those links are working OK, but on docs.microsoft portal they don't make sense as it just appends it to the url.

Reply:
I have a link in the description of this PR where if you click the `IIconProps` link it does nothing